gr-network/lib/udp_sink_impl.h

```diff
@@ -24,6 +24,7 @@ namespace network {
 class NETWORK_API udp_sink_impl : public udp_sink
 {
 protected:
+    const std::string d_host;
```

Comment:
I have come around to thinking that const member variables is more trouble than they're worth, so am leaning towards no-consting this. Other opinions?

Reply:
I think I just copy-pasted that from the constructor. The TCP sink doesn't have it. Removed.